### PR TITLE
feat: add bu1 module

### PR DESCRIPTION
**Pull request by rutalreja-deloitte**

### CODE REVIEW

File: 5-infrastructure/business_unit_1/development/development.auto.tfvars [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+parent_folder              = "953533373736"
+boa_gke_project_id         = "prj-bu1-d-boa-gke-ecb0"
+boa_ops_project_id         = "prj-bu1-d-boa-ops-040f"
+boa_sec_project_id         = "prj-bu1-d-boa-sec-7cd1"
+boa_sql_project_id         = "prj-bu1-d-boa-sql-4f49"
+gcp_shared_vpc_project_id  = "prj-d-shared-base-f12c"
+terraform_service_account  = "terraform-boa-sa@prj-d-shared-base-f12c.iam.gserviceaccount.com"
+gke_cluster_1_cidr_block   = "100.64.78.0/28"
+gke_cluster_2_cidr_block   = "100.65.70.0/28"
+gke_mci_cluster_cidr_block = "100.64.70.0/28"
```
Comment (bharathkkb) [inline 5-infrastructure/business_unit_1/development/development.auto.tfvars:26]: nit: newline

File: 5-infrastructure/business_unit_1/development/variables.tf [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+variable "location_secondary" {
+  type        = string
+  description = "The secondary region for deployment, if not set default locations for each resource are taken from variables file"
+  default     = "us-west1"
+}
+
+variable "gcp_shared_vpc_project_id" {
+  type        = string
+  description = "The host project id of the shared VPC. Can be left blank if prjs deployed follow naming convention (Eg. prj-d-shared-base-xxxx)"
+  default     = ""
```
Comment (bharathkkb) [inline 5-infrastructure/business_unit_1/development/variables.tf:42]: Lets explicitly require these

File: 5-infrastructure/business_unit_1/development/development.auto.tfvars [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+parent_folder              = "953533373736"
+boa_gke_project_id         = "prj-bu1-d-boa-gke-ecb0"
+boa_ops_project_id         = "prj-bu1-d-boa-ops-040f"
+boa_sec_project_id         = "prj-bu1-d-boa-sec-7cd1"
+boa_sql_project_id         = "prj-bu1-d-boa-sql-4f49"
+gcp_shared_vpc_project_id  = "prj-d-shared-base-f12c"
+terraform_service_account  = "terraform-boa-sa@prj-d-shared-base-f12c.iam.gserviceaccount.com"
```
Comment (bharathkkb) [inline 5-infrastructure/business_unit_1/development/development.auto.tfvars:23]: Lets explicitly set all these as examples following example-foundation conventions so it is clear to the user that this needs to me changed.
https://github.com/terraform-google-modules/terraform-example-foundation/blob/b1704781ba4a004de5ba8917feded5361259ce12/3-networks/common.auto.example.tfvars#L19

File: 5-infrastructure/modules/base-env/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Base Env Module
```
Comment (bharathkkb) [inline 5-infrastructure/modules/base-env/README.md:1]: This is in the other PR, lets remove this from here and we can rebase when the other PR is merged

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @rutalreja-deloitte

**Comment by rutalreja-deloitte** [comment]
Ready apart from pull from main once base-env merges, then lint will pass

**Comment by bharathkkb** [comment]
@rutalreja-deloitte other PR is merged

**Pull request closed without merging**